action.yml

```diff
@@ -51,7 +54,7 @@ runs:
           SET_ENV=$(AWS_ACCESS_KEY_ID=${{ inputs.access_key_id }} \
             AWS_SECRET_ACCESS_KEY=${{ inputs.secret_access_key }} \
             AWS_DEFAULT_REGION=${{ inputs.region }} \
-            aws ssm get-parameter --name "${{ inputs.parameter_name }}" --with-decryption --output text --query Parameter.Value --output text \
+            aws ssm get-parameter --name "${{ inputs.parameter_name }}" --with-decryption --output text --query Parameter.Value --output text --version "${{ inputs.parameter_version }}" \ 
```

Comment:
This is close, but what happens when we do not want to specify a version. So because the version should be optional, it should not always be included in the `aws ssm get-parameter` call.